### PR TITLE
New user interface

### PR DESCRIPTION
**Pull request by arjun149**
Work in progress.

### CODE REVIEW

**Comment by ftavella** [comment]
This is the initial layout that we discussed on 10/30/23
app_layout_meeting_10_30_23

**Comment by ftavella** [comment]
Meeting 1/10/24
Finish just the layout without any connection to the model. This means:
1. Invent two random vectors of 100 values called `wakeTime` and `fallAsleepTime`. It is important that these two come from outside of the scope of `renderPlot` because they will ultimately come from the simulation which we want to run only once, outside each of the plots.
2. Then using those two, fill up the histograms.
3. After that process the two to obtain `sleepDuration`.
4. Calculate the average of `wakeTime`, `fallAsleepTime`, and `sleepDuration`.
5. Display them on the bottom right panel

For the countries, simply make a list of three for the user to select.

**Comment by arjun149** [comment]
Franco, the changes have been made - here's an attached screenshot. Let me know the next steps. Thanks!

newappui

File: src/user_interface.R [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+
+)
+
+# Define server logic required to draw a histogram
+server <- function(input, output) {
+    
+    wakeTime <- c()
+    fallAsleepTime <- c()
+    time <- c()
```
Comment (ftavella) [inline src/user_interface.R:72]: Great job defining things outside and using them within the plot functions!

**Comment by arjun149** [comment]
Hey, Franco -

The UI should be updated with the new code. I fixed the axis titles & labels, and the ticks are better now, too. Let me know if we are good to proceed. Thanks!

**Review by ftavella — APPROVED** [review]

**Pull request merged**